src/app/app.component.html

```diff
@@ -25,18 +26,27 @@
       ></plh-template-container>
     </ion-content>
   </ion-menu>
-  <div style="display: flex; flex-direction: column; height: 100%">
-    <plh-main-header></plh-main-header>
-    <div class="route-container">
-      <ion-router-outlet id="main-content"></ion-router-outlet>
+  <!-- Main content: shows in split-pane when sidebar route active -->
+  <ion-split-pane when="lg" contentId="main" [disabled]="!sidebarRouter.isActivated">
```

Comment:
@volloholic - here is the code that handles when/if to use split panes. Confusingly enough _disabling_ the pane means the content will still render (just not in split pane)

It uses `!sidebarRouter.isActivated` which is not actually defined in code, but instead read directly from the router outlet further below
```
<router-outlet name="sidebar" #sidebarRouter="outlet"></router-outlet>
```
Angular has a system where defining a `#id` on any element provides local reference, so this code exposes the underlying `outlet` via the variable `sidebarRouter` for access within the local html, and this is what we are reading in the code above.

By assigning the disabled property to activation state, this means currently the sidebar will always be enabled whenever the routing asserts a sidebar should exist (e.g. `/template/home(sidebar:somePage`) and not whenever there is no corresponding route active. 

There is the additional caveat of `when="lg"` which uses screen size breakpoints to only show on larger screens. All of these things can indeed be customised how we see fit including sizes of breakpoints and sizes of sidebar.
Using this default additionally means that we can still access the sidebar content on smaller devices as a sidemenu instead of sidepane (just like how the left menu can expand out)

Reply:
Thanks I'd missed the when="lg".